### PR TITLE
[bitnami/grafana] Change the test temporaly

### PR DESCRIPTION
**Pull request by Mauraza**
### Description of the change

Change the test temporarily until this will be fixed in https://github.com/grafana/grafana/issues/68213

When this will be fixed this test should be added again:
```
  grafana-cli-data-migration:
    exec: cd /opt/bitnami/grafana && grafana-cli admin data-migration encrypt-datasource-passwords
    exit-status: 0
    stdout:
      - /datasources.*already encrypted/
```
### Applicable issues

- fixes #

### Checklist

- [ ] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [ ] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [X] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [X] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: .vib/grafana/goss/goss.yaml [hunk 1]
```diff
@@ -8,11 +8,6 @@ file:
     exists: true
     mode: "2755"
 command:
-  grafana-cli-data-migration:
-    exec: cd /opt/bitnami/grafana && grafana-cli admin data-migration encrypt-datasource-passwords
-    exit-status: 2
-    stderr:
-      - /duplicate metrics collector registration attempted/
```
Comment (joancafom) [inline .vib/grafana/goss/goss.yaml:15]: I'd say we should not remove the test, but rather disable/comment it out (if disabling it is not possible). Additionally, I would also take this opportunity to fix the test, which currently produces a false positive.

**Review by joancafom — CHANGES_REQUESTED** [review]
Thanks for your change! Please, take a look at my comment when you have a chance :)

File: .vib/grafana/goss/goss.yaml (hunk 1 above)
Comment (Mauraza) [reply .vib/grafana/goss/goss.yaml]: I changed the test, tell me if that it's okay with you.

File: .vib/grafana/goss/goss.yaml [hunk 2]
```diff
@@ -9,10 +9,12 @@ file:
     mode: "2755"
 command:
   grafana-cli-data-migration:
-    exec: cd /opt/bitnami/grafana && grafana-cli admin data-migration encrypt-datasource-passwords
-    exit-status: 2
-    stderr:
-      - /duplicate metrics collector registration attempted/
+    # The 'grafana-cli admin data-migration encrypt-datasource-passwords' it's not working meanwhile this will be fixed
```
Comment (joancafom) [inline .vib/grafana/goss/goss.yaml:12]:
```suggestion
    # The 'grafana-cli admin data-migration encrypt-datasource-passwords' command is not working. In the meantime it is fixed,
```

File: .vib/grafana/goss/goss.yaml [hunk 3]
```diff
@@ -9,10 +9,12 @@ file:
     mode: "2755"
 command:
   grafana-cli-data-migration:
-    exec: cd /opt/bitnami/grafana && grafana-cli admin data-migration encrypt-datasource-passwords
-    exit-status: 2
-    stderr:
-      - /duplicate metrics collector registration attempted/
+    # The 'grafana-cli admin data-migration encrypt-datasource-passwords' command is not working. In the meantime it is fixed,
+    # the test checks the 'encrypt-datasource-passwords' appears.
```
Comment (joancafom) [inline .vib/grafana/goss/goss.yaml:13]:
```suggestion
    # the test checks the 'encrypt-datasource-passwords' option appears.
```

**Pull request merged**